src/dagre-js/create-edge-paths.js

```diff
@@ -3,9 +3,9 @@ import _ from 'lodash-es';
 import { intersectNode } from "./intersect/intersect-node";
 import * as util from "./util";
 
-export { createEdgePaths };
+export { createEdgePaths, setCreateEdgePaths };
 
-function createEdgePaths(selection, g, arrows) {
+var createEdgePaths = function(selection, g, arrows) {
```

Comment:
Do you mind using `const` instead of `var`?

Reply:
https://github.com/tbo47/dagre-es/pull/3#discussion_r1014727984